contentcuration/contentcuration/tests/custom_pytest_db_backend/base.py

```diff
@@ -0,0 +1,32 @@
+from django.db.backends.postgresql.base import DatabaseWrapper as PostgresDatabaseWrapper
+from django.db.backends.postgresql.creation import DatabaseCreation as PostgresDatabaseCreation
+
+
+class CustomDBCreationForPytest(PostgresDatabaseCreation):
+    """
+    Overriding creation module to enable postgres pgvector extension before
+    pytest runs migration. Because embeddings table rely on pgvector
+    extension to work.
+    """
+
+    def _create_test_db(self, verbosity, autoclobber, keepdb=False):
+        from contentcuration.management.commands.setup import enable_pgvector_extension
+
+        # Create test database and get its name.
+        test_db_name = super()._create_test_db(verbosity, autoclobber, keepdb)
```

Comment:
Ah @vkWeb if the super returns the database name, seems like this override should too!?

Reply:
Yeah right, that's an amazing point! ideally it should return `test_db_name`. I skipped returning because `create_test_db` the function that utilizes `_create_test_db` doesn't used its return value. But yes, returning test db name would be an ideal behaviour. Sending a commit.